mibs/lenovoxcc/lnvgy_fw_xcc_tei3c0n-4.83_lenovosmi-anyos_noarch.mib

```diff
@@ -0,0 +1,72 @@
+LENOVO-SMI-MIB DEFINITIONS ::= BEGIN
```

Comment:
Same here : 
`LENOVO-SMI-MIB`

and same for both 2 remaning MIB files

Reply:
Done!